StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/USBankAccount/InstantDebitsPaymentMethodElement.swift

```diff
@@ -11,7 +11,10 @@
 @_spi(STP) import StripeUICore
 import UIKit
 
-final class InstantDebitsPaymentMethodElement: Element {
+final class InstantDebitsPaymentMethodElement: ContainerElement {
```

Comment:
This code is a copy from here:

https://github.com/stripe/stripe-ios/pull/3592/files#diff-fd0e20ad72dcc1cb100530b2d43db395c8e12caa20dff61ba277af81be0a2189R14-R206

![Screenshot 2024-05-20 at 1 55 44 PM](https://github.com/stripe/stripe-ios/assets/105514761/2cee2ec1-4fbf-4611-a46a-d691ab770070)